.devcontainer/scipy2023/tasks.json

```
@@ -0,0 +1,16 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "jupyterlab",
+      "type": "shell",
+      "command": "/srv/conda/envs/notebook/bin/jupyter lab",
```

```suggestion
      "command": "/srv/conda/envs/notebook/bin/jupyter lab --no-browser",
```
i think the no-browser is typical when launching on a remote server